src/rmp/src/Restructure.cpp

```diff
@@ -254,16 +222,16 @@ void Restructure::runABC()
       if (writeAbcScript(abc_script_file)) {
 	// Call linked abc
 	Abc_Frame_t * pAbc;
-	char Command[1000];
+	std::string Command = "";
 
 	Abc_Start();
 	pAbc = Abc_FrameGetGlobalFrame();
```

Comment:
Abc_Frame_t* pAbc = Abc_FrameGetGlobalFrame();

I guess pAbc  came from the abc example but it is a poor name (we don't use Hungarian notation).  abc_frame perhaps or just abc.